File: 8kyu/(8 kyu) Plural/(8 kyu) Plural.py
# 1 Simple solution
def plural(n):
    if n == 1 or n == -1:
        return False
    else:
        return True

# 2 Optimized solution
def plural(n):
    return n != 1 and n != -1

# 3 Clever solution
plural=lambda n:n!=1

# 4 Coding golf
plural = 1..__ne__

File: 8kyu/(8 kyu) Plural/(8 kyu) Plural.rs
// 1 Simple solution
fn plural (n: f64) -> bool {
    if n == 1.0 || n == -1.0 {
        false
    } else {
        true
    }
}

// 2 Optimized solution
fn plural(n: f64) -> bool {
    n != 1.0 && n != -1.0
}

// 3 Clever solution
fn plural(n: f64) -> bool {
    n.abs() != 1.0
}

// 3 Coding golf
fn plural(n: f64) -> bool {n != 1.}
